feat: add contributing guide + add "translate" script for quick template (#7)
mattpocock/ts-error-translator: CONTRIBUTING.md

> # How do I Contribute 💪

Reviewer: ```suggestion
# How do I contribute? 💪
```

2. Install [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#windows-stable) (for installing npm dependencies, using yarn workspaces)

## How to start developing?

Clone the repo and install the needed dependencies for all the packages by following these steps:

```sh
git clone https://github.com/mattpocock/ts-error-translator.git
cd ts-error-translator
yarn
yarn dev # This will run the next app
```

## Adding/editing errors' translations

You'll find all of the errors' translations at `packages/engine/errors` and they follow the following conventions:

- Every file must be named with its error code: `<code>.md` (cause how it works is that there's a regex expression that matches your error message with the
defined ones and get the error code back then search for `<code>.md` to read the explanation for it).

> 💡 Hint: To find the code for the error you're looking for, have a look inside [`tsErrorMessages.json`](https://github.com/mattpocock/ts-error-translator/blob/main/packages/engine/src/tsErrorMessages.json),
> or check your console, you'll see this formatting `error TS<code>: <msg>`.

> - You have to follow our template by running the following command. This will write an example file at `packages/engine/errors/<code>.md` with placeholders on how the explanation should be written.

Reviewer: ```suggestion
- You can follow our template by running the following command. This will write an example file at `packages/engine/errors/<code>.md` with placeholders on how the explanation should be written.
```
